lib/runner/folder-walk.js

```diff
@@ -215,7 +215,7 @@ class Walker {
             return this.readFolderDeep(path.join(folderPath, resource));
           }
 
-          if (stat.isFile() && Utils.isFileNameValid(resource)) {
+          if (stat.isFile() && Utils.isFileNameValid(resource) && !(this.settings.disable_typescript === true && Utils.isTsFile(resource))) {
```

Comment:
maybe this should be a small local method, like shouldLoadFile or something.

Reply:
moved it above to make it more cleaner